sechub-pds-solutions/iac/helm/pds-iac/Chart.yaml

```
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: MIT
+
+apiVersion: v2
+name: pds-iac
+description: SecHub PDS + IaC tools as Helm chart for Kubernetes
+
+type: application
+
+maintainers:
+  - name: Jeremias Eppler
+  - name: Rouven Haertel
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

If everything works from a Helm perspective, then we could go with version 1.0.0
```suggestion
version: 1.0.0
```